Source/Scene/Cesium3DTileBatchTable.js

```diff
@@ -1231,56 +1229,58 @@ define([
         OPAQUE_AND_TRANSLUCENT : 2
     };
 
-    function updateDerivedCommands(derivedCommands, command) {
-        for (var name in derivedCommands) {
-            if (derivedCommands.hasOwnProperty(name)) {
-                var derivedCommand = derivedCommands[name];
-                derivedCommand.castShadows = command.castShadows;
-                derivedCommand.receiveShadows = command.receiveShadows;
-                derivedCommand.primitiveType = command.primitiveType;
-            }
+    Cesium3DTileBatchTable.prototype.addDerivedCommands = function(frameState, commandStart) {
+        if (!frameState.passes.render) {
+            return;
+        }
+
+        var commandList = frameState.commandList;
+        var commandLength = commandList.length;
+        if (commandStart === commandLength) {
+            return;
         }
-    }
 
-    Cesium3DTileBatchTable.prototype.getAddCommand = function() {
-        var styleCommandsNeeded = getStyleCommandsNeeded(this);
         var tile = this._content._tile;
         var tileset = tile._tileset;
+        var bivariateVisibilityTest = tileset.skipLODs && tileset._hasMixedContent && frameState.context.stencilBuffer;
+        var styleCommandsNeeded = getStyleCommandsNeeded(this);
```

Comment:
Is style the right semantic here since these derived commands could also be for the bivariate visibility test?

Perhaps just `derivedCommandsNeeded` is more appropriate.